Devscord.DiscordFramework/ReactionContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Discord;
+using Discord.WebSocket;
+
+namespace Devscord.DiscordFramework
+{
+    public class ReactionContext
+    {
+        public ISocketMessageChannel Channel { get; }
+        public IEmote Emote { get; }
```

Comment:
co to jest IEmote

Reply:
https://discord.foxbot.me/docs/api/Discord.IEmote.html?q=IEmote